rmw_implementation/package.xml

```diff
@@ -11,11 +11,20 @@
 
   <build_depend>rcutils</build_depend>
   <build_depend>rmw</build_depend>
+  <build_depend>rmw_connext_cpp</build_depend>
+  <build_depend>rmw_fastrtps_cpp</build_depend>
+  <build_depend>rmw_opensplice_cpp</build_depend>
 
   <depend>libpoco-dev</depend>
   <depend>poco_vendor</depend>
   <depend>rmw_implementation_cmake</depend>
-  <depend>rmw_fastrtps_cpp</depend>
+  <!--
+  Rather than a hard-dependency on any one rmw implementation.
+  The debian/control template for this package will be modified to depend on either
+  rmw_fastrtps_cpp or rmw_connext_cpp or rmw_opensplice_cpp with rmw_fastrtps_cpp being
+  the default choice if no other rmw implementation is currently installed.
+  <depend>rmw_fastrtps_cpp</depend> 
```

Comment:
Should I comment this out since there won't be an absolute dependency on rmw_fastrtps_cpp or leave it in for from-source builds?

Reply:
Since the Debian package shouldn't have a hard dependency on FastRTPS it shouldn't be commented in.

I am not sure if the comment here is a good idea. It solely describes packaging customizations. Having them duplicated here just adds the risk of them getting out-of-sync while not adding much value here.

Reply:
> Since the Debian package shouldn't have a hard dependency on FastRTPS it shouldn't be commented in.

I don't quite follow the outcome of this sentence. Should I restore the dependency which will be modified rather than removed at packaging time or leave it commented  while modifying the comment above it to reduce bitrot?

Reply:
I would remove this while comment since it is packaging specific.